tomopy/recon/rotation.py

```diff
@@ -191,7 +193,7 @@ def _find_center_cost(
     center = np.array(center, dtype='float32')
     rec = recon(
         tomo_ind, theta, center,
-        sinogram_order=sinogram_order, algorithm='gridrec')
+        sinogram_order=sinogram_order, algorithm=algorithm, filter_name=filter_name)
```

Comment:
Sane here: ```algorithm='gridrec'```.

Reply:
@dgursoy  this was a change I did with @vdeandrade long time ago .... I think is better to have a default as you did. @vdeandrade: any comment?